android/src/main/java/com/tailscale/ipn/ui/model/IpnState.kt

```diff
@@ -5,114 +5,15 @@ package com.tailscale.ipn.ui.model
 
 import kotlinx.serialization.Serializable
 
-class IpnState {
```

Comment:
We don't use this now but there are API endpoints that are yet to be implemented that will require us to fetch PeerStatus (for example). 

Removing this doesn't really gain us much.  Commenting out the unused fields will.